spectral_cube/tests/test_spectral_cube.py

```diff
@@ -74,18 +75,31 @@ def test_huge_disallowed():
 
     cube, data = cube_and_raw('vda_Jybeam_lower.fits')
 
-    data = np.empty([1e2,1e3,1e3])
     cube = SpectralCube(data=data, wcs=cube.wcs)
 
-    assert cube._is_huge
+    assert not cube._is_huge
+    
+    # We need to reduce the memory threshold rather than use a large cube to
+    # make sure we don't use too much memory during testing.
```

Comment:
I think we still need some genuinely large cube tests, though perhaps not to be run regularly on travis... that may be a different issue, though.  See, e.g., #259, which was uncaught because we don't have automated tests on gigantocubes